jerry-core/debugger/debugger.c

```diff
@@ -38,6 +49,11 @@
   type *name_p = ((type *) recv_buffer_p)
 
 /**
+ * Sleep time in milliseconds between each jerry_debugger_receive() call
```

Comment:
style: either don't add the parentheses after jerry_debugger_receive or use jerry coding style there as well and add a space between the function name and the parentheses